crates/swc_ecma_minifier/src/pass/mangle_names/preserver.rs

```diff
@@ -104,7 +116,7 @@ impl Visit for Preserver {
         n.visit_children_with(self);
 
         if let Pat::Ident(i) = n {
-            if self.should_preserve {
+            if self.should_preserve || self.is_reserved(&i.id) {
```

Comment:
Question: Do we think reserved identifiers should apply to private identifiers too?

Reply:
I don't think so, but I prefer following behavior of terser.

Reply:
I checked the behaviour of terser and `reserved` and `properties.reserved` do not apply to private props so removed this logic